documentation/book/con-what-the-cluster-operator-does.adoc

```diff
@@ -8,7 +8,7 @@
 
 {ProductName} uses the Cluster Operator to deploy and manage clusters for:
 
-* Kafka (including Zookeeper)
+* Kafka (including Zookeeper and Kafka Exporter)
```

Comment:
Should the Entity Operator be there? @scholzj

Reply:
I'm fine both ways. It can be mentioned there but I do not necessarily see it as an issue when it is not there.